pennylane/operation.py

```diff
@@ -3152,7 +3152,10 @@ def convert_to_legacy_H(op):
     else:
         raise ValueError("Could not convert to Hamiltonian. Some or all observables are not valid.")
 
-    return qml.Hamiltonian(coeffs, ops)
+    with disable_new_opmath_cm():
+        hamiltonian = qml.Hamiltonian(coeffs, ops)
+
+    return hamiltonian
```

Comment:
Nice! I already pushed this to `ham-tests`, glad we came to the same conclusion

Reply:
I just called it `res` instead and took that over now to not have to solve that merge conflict to ham-tests